src/main/react-front-end/src/model/reducers.js

```diff
@@ -12,8 +12,8 @@ queue
 import { LOGIN, LOGOUT, PROMOTE, ENDPOINT_PROGRESS, ENDPOINT_UPDATE, UPDATE_HASH } from './actions';
 import { transferOptimizations } from "./actions";
 
+// import {cookie} from "react-cookies";
```

Comment:
@aashish-jain this can be removed for now.

Reply:
Removed